Gem/Code/Source/HighInstanceExampleComponent.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <EntityLatticeTestComponent.h>
+#include <Utils/ImGuiSidebar.h>
+#include <Utils/ImGuiAssetBrowser.h>
+#include <AzCore/Component/TickBus.h>
+
+namespace AtomSampleViewer
+{
+    /*
+        This test loads a configurable lattice of entities with randomized meshes and materials.
+
+        The assets that are applied to the entities are chosen from a configurable 
+        "allow-list" of models and materials.
```

Comment:
This comment seems need to be updated to reflect the purpose of this sample. Otherwise it doesn't have much different than the asset load test sample.